dataverse-persistence/src/test/java/edu/harvard/iq/dataverse/persistence/ScriptRunner.java

```diff
@@ -0,0 +1,100 @@
+package edu.harvard.iq.dataverse.persistence;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.io.Reader;
+import java.io.StringReader;
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+/**
+ * Runner of sql scripts
+ * <p>
+ * Based on MyBatis2 ScriptRunner
+ * 
+ * @see https://github.com/mybatis/ibatis-2/blob/master/src/main/java/com/ibatis/common/jdbc/ScriptRunner.java
+ */
+@Stateless
+public class ScriptRunner {
+    private static final Logger LOG = Logger.getLogger(ScriptRunner.class.getCanonicalName());
+    private static final String DELIMITER = ";";
+
+
+    @PersistenceContext(unitName = "VDCNet-ejbPU")
+    private EntityManager entityManager;
+
+
+    // -------------------- LOGIC --------------------
+
+    /**
+     * Runs an SQL script from classpath
+     */
+    public void runScriptFromClasspath(String classpath) {
+        try (Reader reader = new StringReader(IOUtils.resourceToString(classpath, StandardCharsets.UTF_8))) {
+            LOG.info("Run script from classpath: " + classpath);
+            runScript(reader);
+        } catch (IOException e) {
+            throw new RuntimeException("Encountered error while reading sql file", e);
+        }
+    }
+
+    /**
+     * Runs an SQL script (read in using the Reader parameter)
+     * @throws IOException 
+     */
+    private void runScript(Reader reader) throws IOException {
+        StringBuffer command = null;
+        LineNumberReader lineReader = new LineNumberReader(reader);
+
+        try {
+            String line = null;
+            while ((line = lineReader.readLine()) != null) {
+                if (command == null) {
```

Comment:
Wouldn't it make sense to just initialize StringBuffer() at the beginning  instead of checking since we know it's gonna be null?

Reply:
good point